website/source/docs/builders/openstack.html.md

```diff
@@ -127,6 +127,8 @@ builder.
     after creation. An image member is usually a project (also called the
     "tenant") with whom the image is shared.
 
+-   `image_min_disk` (int) - Minimum disk of needed to boot image size in GB. 
```

Comment:
Maybe this from the cli docs:

_Minimum disk size needed to boot image, in gigabytes_